dashboard/app/models/activity_section.rb

```diff
@@ -51,11 +51,27 @@ def summarize
       duration: duration,
       remarks: remarks,
       description: Services::I18n::CurriculumSyncUtils.get_localized_property(self, :description),
-      tips: tips,
+      tips: localized_tips,
       progressionName: localized_progression_name
     }
   end
 
+  # Translates the content of tips in the adequate format.
+  # @attr [Array<Hash>] tips each Hash contains a "type" and a "markdown". get_localized_property returns the same
+  # [Array<Hash>] when the locale is the default_locale, otherwise, it returns or Array of to translated "markdown"
+  # content [Array<String or NilClass>].
+  #
+  # @return [Array <Hash>] copy of tips containing translated content
+  def localized_tips
+    return nil unless tips
+    local_tips = Services::I18n::CurriculumSyncUtils.get_localized_property(self, :tips)
+    tips_clone = tips.map(&:clone)
+    tips_clone.each_with_index do |tip, index|
+      tip["markdown"] = local_tips[index].dup unless (local_tips[index] == tip) || local_tips[index].nil?
```

Comment:
`local_tips[index]` is a frozen string, which results in an error when  `Services::MarkdownPreprocessor.process!(tip["markdown"])` tries to parse the markdown text.
Normally, `tip["markdown"]` is not frozen, for that reason I duplicated local_tips when updating `tip["markdown"]` content.